app/javascript/stimulus/__tests__/controllers/buffer_controller.js

```diff
@@ -0,0 +1,52 @@
+import { Application } from 'stimulus';
```

Comment:
I think this should be renamed to `buffer_controller.test.js`

Reply:
Good catch!